aerleon/lib/cisco.py

```diff
@@ -805,7 +821,7 @@ class ObjectGroupTerm(Term):
     in the acl.
     """
 
-    def _FormatPort(self, port, proto):
+    def _FormatPort(self, port: Union[Tuple[()], Tuple[int, int]], proto: str) -> str:
```

Comment:
Same as above, should be `Optional[Tuple[int, int]]`

Reply:
Same response as above. Lets leave the conversation there and we can resolve this based on the one above.